apps/meteor/ee/server/apps/communication/rest.ts

```diff
@@ -1124,14 +1128,48 @@ export class AppsRestApi {
 						return API.v1.failure('Invalid status provided, it must be "status" field and a string.');
 					}
 
-					const prl = manager.getOneById(this.urlParams.id);
+					const baseUrl = orchestrator.getMarketplaceUrl();
```

Comment:
This variable, the `headers` one on line 1135, `version` on line 1144, `isStoredAppEnterpriseOnly` on line 1146 and `isAppEnterpriseOnly` on line 1148 are all used _exclusively_ inside the if statement that start on line 1150 - declaring them outside of that block makes code harder to read in the case they are irrelevant and they consume resources even when they're not used.

Please move them into the scope where they're used